svelte/fr/tutorial/05-events/01-dom-events/app-a/App.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+	let m = { x: 0, y: 0 };
+
+	function handleMousemove(event) {
+		m.x = event.clientX;
+		m.y = event.clientY;
+	}
+</script>
+
+<div>
+	La souris se trouve à la position ({m.x} ; {m.y})
```

Comment:
```suggestion
	Le curseur se trouve à la position ({m.x} ; {m.y})
```
Souris / curseur.
Difficile de choisir, en anglais aussi ils jonglent entre _mouse_ et _cursor_